tstdc/README.md

```diff
@@ -0,0 +1,21 @@
+# MobileCoin: Rust wrappers around SGX synchronization primitives
+
+[![Project Chat][chat-image]][chat-link]<!--
+-->![License][license-image]<!--
+-->![Target][target-image]<!--
+-->[![Crates.io][crate-image]][crate-link]<!--
+-->[![Docs Status][docs-image]][docs-link]<!--
+-->[![Dependency Status][deps-image]][deps-link]
+
+Rust wrappers around SGX synchronization primitives
```

Comment:
Up for suggestions on this description, kind of struggled here as the library name (and thus the crate name) feels a bit odd

Reply:
LGTM